ibis/backends/tests/test_udf.py

```diff
@@ -27,6 +29,9 @@
 @mark.notimpl(["pyspark"])
 @mark.notyet(["datafusion"], raises=NotImplementedError)
 def test_udf(batting):
+    if sys.version_info >= (3, 11):
+        xfail("Docker image has Python 3.10, results in `cloudpickle` version mismatch")
```

Comment:
This is going to xfail for all backends. Can you adjust so that it's only the flink backend that fails here?

The easiest way is probably to use

```python
@broken(["flink"], strict=False, reason="cloudpickle shenanigans")
```

Reply:
Oh, oops.

Reply:
Decided it would be cleaner to just pass a `condition` for the `cloudpickle` shenanigans marker, and that way I could cleanly separate those (failing on 3.11) from other failures (on 3.9), and be as explicit as possible.